app/models/piece.rb

```diff
@@ -1,3 +1,10 @@
 class Piece < ApplicationRecord
   belongs_to :game
 end
+
+class Pawn < Piece; end
+class Knight < Piece; end
+class Bishop < Piece; end
+class Rook < Piece; end
+class Queen < Piece; end
+class King < Piece; end
```

Comment:
Can you create a file for each of these classes?

For example": `models/pawn.rb`

We'll be adding a lot of code to them soon, so having them in separate files will make it more organized.

😄